### PR TITLE
Convert to Comprehensive Theming

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@caesar2164

File: lms/templates/theme-header.html [hunk 1]
```diff
@@ -25,17 +26,30 @@
...
 
 ## When authenticated, show both the about page and find courses
 <%block name="navigation_global_links_authenticated">
   <li class="nav-global-01">
-    <a href="${marketing_link('ABOUT')}">${_('About')}</a>
+    <a class="btn" href="${marketing_link('ABOUT')}">${_('About')}</a>
   </li>
   <li class="nav-global-02">
-    <a href="${marketing_link('COURSES')}">${_('Find Courses')}</a>
+    <a class="btn" href="/courses">${_('Explore Courses')}</a>
```
Comment (stvstnfrd) [inline lms/templates/theme-header.html]: You can't hard-code urls.

**Comment by stvstnfrd** [comment]
Closing in favor of https://github.com/Stanford-Online/openedx-themes

**Comment by SamuelMarks** [comment]
Hmm… so is that following comprehensive theming? - Is it compatible with the latest [vanilla] OpenEdX?

**Comment by caesar2164** [comment]
@SamuelMarks I would NOT suggest using this as an example of a comprehensive theme as we converted from Stanford theming so we have some bizarre things set up to keep past functionality...

**Pull request closed without merging**